A2017/tensorflow.ipynb

```diff
@@ -0,0 +1,892 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# TensorFlow tutorial\n",
+    "_MILA, November 2017_\n",
+    "\n",
+    "## Protip: browsing through the TensorFlow API\n",
+    "\n",
+    "The [devdocs.io](http://devdocs.io/) website is an amazing resource to browse through the TensorFlow Python API (as well as many other APIs such as numpy or the Python API itself).\n",
+    "\n",
+    "# Using TensorFlow at MILA\n",
+    "\n",
+    "The most straightforward way to access TensorFlow using the MILA software stack is through the `TODO: WRITEME` conda environment. To activate the `TODO: WRITEME` environment, use the command\n",
+    "\n",
+    "```bash\n",
+    "source activate TODO: WRITEME\n",
+    "```\n",
+    "\n",
+    "To return back to normal, simply use the bash command\n",
+    "\n",
+    "```bash\n",
+    "source deactivate\n",
+    "```\n",
+    "\n",
```

Comment:
I think it miss a section to explain how to install tensorflow (e.g. for users who want to use it in their own computers). I suggest to just describe the pip command:
```
pip install tensorflow-gpu
```
https://www.tensorflow.org/install/install_windows

Reply:
Done via 5c29d47.